src/components/utilities/dialog.tsx

```diff
@@ -72,7 +72,9 @@ export class DialogComponent extends BaseComponent<IProps> {
                 {Icon && <Icon />}
               </div>
               <div className="dialog-title-text">{title}</div>
-              <button onClick={this.handleCancelDialog}><CloseIcon /></button>
+              <div className="dialog-title-cancel">
+                <button onClick={this.handleCancelDialog}><CloseIcon /></button>
```

Comment:
This button should have some[ accessible text](https://dequeuniversity.com/rules/axe/4.10/button-name) inside it.

Passing since this bug is pre-existing, but it should be an easy fix.